src/components/molecules/WalletItem/walletItem.molecule.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {CONNECTOR_METADATA} from "../../../shared/constants/web3.constants";
+import {EColor, EFontWeight} from "../../../shared/types/styles.types";
+import Typography from "../../atoms/Typography/typography.atom";
+import {IWalletItem} from "./walletItem.molecule.types";
+import styles from "./walletItem.molecule.module.scss";
+
+const WalletItem: React.FC<IWalletItem> = ({ connectorType, onClick }) => {
```

Comment:
Hmm, can we keep the single param "props" notation throughout the codebase and destructure it in new line.
My rationale here is that it's cleaner to have the destructuring in new lines and not make a really long line with the function definition. While also keeping the props keyword from back in the days React before FC component were introduced. Wdyt?

Reply:
Sure! +1 for consistency